libdrgn/register_state.c

```diff
@@ -31,6 +31,21 @@ struct drgn_register_state *drgn_register_state_create_impl(uint32_t regs_size,
 	return regs;
 }
 
+struct drgn_register_state *drgn_register_state_dup(
+		const struct drgn_register_state *regs)
```

Comment:
Style nit: let's format it like this:

```suggestion
struct drgn_register_state *
drgn_register_state_dup(const struct drgn_register_state *regs)
```